lib/ui/widgets/concord_tiles.dart

```diff
@@ -0,0 +1,245 @@
+import 'package:flutter/material.dart';
+import 'package:multi_image_picker/multi_image_picker.dart';
+
+class CommunityBulletinTile extends StatelessWidget {
+  final Asset image;
+  final String title;
+  final String description;
+  final Color titleColor;
+  final Color descriptionColor;
+
+  const CommunityBulletinTile({
+    Key key,
+    // TODO:  set placeholder image if one not provided
+    this.image,
+    @required this.title,
+    this.description,
+    this.titleColor,
+    this.descriptionColor
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    double radius = 4.0;
+    double height = 98;
+
+    return Container(
+      //if there is an image, the tile will be longer to accommodate the thumb nail image
+      width: image != null ? MediaQuery.of(context).size.width-16 :  MediaQuery.of(context).size.width-16-height,
+      height: height,
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(radius),
+        color: Color(0xffFAFAFA),
+        boxShadow: [
+          BoxShadow(
+            color: Colors.grey.withOpacity(0.6),
+            spreadRadius: 5,
+            blurRadius: 7,
+            offset: Offset(0,3)
+          )
+        ]
+      ),
+      child: image != null ?
+        //if there is an image
+        Row(
+          children: <Widget>[
+            AssetThumb(
+              asset: image,
+              width: height.floor(),
+              height: height.floor(),
+            ),
+            _textPortion()
+          ],
+        ) : 
+        //if there is no image
```

Comment:
See the change to line 43.

```suggestion
        //if there is no image
```